dotcom-rendering/src/amp/components/Analytics.tsx

```diff
@@ -29,6 +29,27 @@ export const Analytics = ({
 }: Props) => {
 	const scripts: string[] = [
 		`<amp-analytics config="https://ophan.theguardian.com/amp.json" data-credentials="include" ></amp-analytics>`,
+		// The componentEvent is the encoded version of componentObject using https://www.urlencoder.org/
+		`<amp-analytics data-block-on-consent>
+			<script type="application/json">
+      {
+        "requests": {
+          "pageviewWithConsent": "\${additionalBase}&componentEvent=\${component}"
+        },
+        "triggers": {
+          "trackPageview": {
+            "on": "visible",
+            "request": "pageviewWithConsent",
+            "vars" : {
+              "additionalBase": "https://ophan.theguardian.com/img/2?viewId=\${ophanPageViewId}&ampViewId=\${pageViewId64}",
```

Comment:
Does additionalBase have to be defined here - as it is defined amp.json? (https://github.com/guardian/ophan/blob/ccaa57bcee3f5f3f83ec28973074c9b3f98f1438/tracker/app/views/amp.scala.json)